content/pages/vocational-rehab-and-employment/apply-vre.md

```diff
@@ -48,59 +33,63 @@ You may also qualify for VR&amp;E benefits if you're a Veteran who was discharge
 
 </div>
 
-<div markdown="1">
-
 ### What kind of VR&amp;E benefits can I get?
 
 - Job search and placement assistance
 - Vocational (job) training 
 - Education resources and counseling
 - Adaptive-housing counseling and referrals
 
-</div>
-
-<div markdown="1">
-
 <hr>
 
 ### How do I get these benefits?
-</div>
+
 You'll need to apply for benefits. Follow these steps to apply:
 
-<ol class="process" markdown="0">
-<li class="step one wow fadeIn animated" markdown="1">
+<ol class="process">
+
+<li class="step one">
+
```

Comment:
The reason the markdown wasn't formatting correctly here - it needs an extra empty line above and below the text.